src/main/fc/config.c

```diff
@@ -257,21 +257,21 @@ void pgResetFn_pwmConfig(pwmConfig_t *pwmConfig)
 
 void pgResetFn_statusLedConfig(statusLedConfig_t *statusLedConfig)
 {
-    for (int i = 0; i < LED_NUMBER; i++) {
-        statusLedConfig->ledTags[i] = IO_TAG_NONE;
+    for (int i = 0; i < STATUS_LED_NUMBER; i++) {
```

Comment:
config is set to zeros before calling reset function. And IO subsystem is designed so that zero is default/NONE. No need to clear it explicitly here .. 
(it is mainly important when using initializers - unspecified members are zero)